package.json

```diff
@@ -53,22 +53,22 @@
   },
   "homepage": "https://github.com/MattiasBuelens/web-streams-polyfill#readme",
   "devDependencies": {
-    "@microsoft/api-extractor": "^7.7.0",
-    "@types/node": "^12.12.17",
-    "@typescript-eslint/eslint-plugin": "^2.11.0",
-    "@typescript-eslint/parser": "^2.11.0",
-    "eslint": "^6.7.2",
+    "@microsoft/api-extractor": "^7.7.8",
+    "@rollup/plugin-inject": "^4.0.1",
+    "@rollup/plugin-replace": "^2.3.1",
+    "@rollup/plugin-strip": "^1.3.2",
+    "@rollup/plugin-typescript": "^3.0.0",
+    "@types/node": "^12.12.28",
+    "@typescript-eslint/eslint-plugin": "^2.20.0",
+    "@typescript-eslint/parser": "^2.20.0",
+    "eslint": "^6.8.0",
     "jasmine": "^3.5.0",
     "micromatch": "^4.0.2",
-    "rollup": "^1.27.10",
-    "rollup-plugin-inject": "^3.0.2",
-    "rollup-plugin-replace": "^2.2.0",
-    "rollup-plugin-strip": "^1.2.2",
-    "rollup-plugin-terser": "^5.1.3",
-    "rollup-plugin-typescript2": "^0.25.3",
-    "ts-morph": "^6.0.2",
-    "tslib": "^1.10.0",
-    "typescript": "^3.7.3",
+    "rollup": "^1.31.1",
+    "rollup-plugin-terser": "^5.2.0",
+    "ts-morph": "^6.0.3",
+    "tslib": "^1.11.0",
+    "typescript": "^3.8.2",
     "wpt-runner": "^2.8.0"
```

Comment:
[wpt-runner version 3.0.0](https://github.com/domenic/wpt-runner/releases/tag/v3.0.0) has recently been released, but it requires Node 10 or higher. This would be a breaking change, so I'm holding off for now.